packages/next/src/server/app-render/app-render.tsx

```diff
@@ -546,10 +546,11 @@ async function getRSCPayload(
     b: ctx.renderOpts.buildId,
     p: ctx.assetPrefix,
     c: url.pathname + url.search,
-    i: couldBeIntercepted,
+    i: !!couldBeIntercepted,
     f: [[initialTree, seedData, initialHead]],
     m: missingSlots,
     G: GlobalError,
+    s: !!ctx.renderOpts.postponed,
```

Comment:
Prefer instead:

```suggestion
    s: typeof ctx.renderOpts.postponed === "string",
```

Reply:
👍 Should we be doing the same [here](https://github.com/vercel/next.js/blob/canary/packages/next/src/server/base-server.ts#L3143-L3145)?